File: go/codewars/8kyu/opposites_attract.go
package main

import "fmt"

// https://www.codewars.com/kata/555086d53eac039a2a000083/train/go

func LoveFunc(flower1, flower2 int) bool {
	return (flower1+flower2)%2 == 1
}

func main() {
	fmt.Println(LoveFunc(1, 4) == true)
	fmt.Println(LoveFunc(2, 2) == false)
	fmt.Println(LoveFunc(0, 1) == true)
	fmt.Println(LoveFunc(0, 0) == false)
}

File: ruby/problem_solving/codewars/8kyu/opposites_attract.rb
def lovefunc( flower1, flower2 )
  !(flower1.odd? ^ flower2.even?)
end

puts lovefunc(1,4) == true
puts lovefunc(2,2) == false
puts lovefunc(0,1) == true
puts lovefunc(0,0) == false
puts lovefunc(5,5) == false